autocrop/cli.py

```diff
@@ -17,7 +17,7 @@
 INPUT_FILETYPES = COMBINED_FILETYPES + [s.upper() for s in COMBINED_FILETYPES]
 
 
-def output(input_filename, output_filename, image):
+def output(input_filename, output_filename, image, basename_noext, extension):
```

Comment:
There's already an `output_filename` variable here, which we can overwrite in the `main` function instead. Cleaner structure!

Reply:
Done, you was right.